src/test/java/de/tum/in/www1/artemis/AtheneIntegrationTest.java

```diff
@@ -102,17 +101,17 @@ public void testProcessingClusterAddedDistances() throws Exception {
 
         for (int clusterIndex = 0; clusterIndex < clusters.size(); clusterIndex++) {
             TextCluster cluster = clusters.get(clusterIndex);
-            assertThat(cluster.size(), equalTo(3));
-            assertThat(cluster.getBlocks(), hasSize(3));
+            assertThat(cluster.size()).isEqualTo(3);
+            assertThat(cluster.getBlocks()).hasSize(3);
 
             List<TextBlock> blocks = cluster.getBlocks();
             for (int blockIndex = 0; blockIndex < blocks.size(); blockIndex++) {
                 TextBlock block = blocks.get(blockIndex);
-                assertThat(block.getAddedDistance(), greaterThan(1.65));
+                assertThat(block.getAddedDistance()).isGreaterThan(1.65);
                 Segment segment = atheneResponse.getClusters(clusterIndex).getSegmentsList().get(blockIndex);
-                assertThat(block.getId(), is(equalTo(segment.getId())));
+                assertThat(block.getId()).isEqualTo(segment.getId());
                 var positionInCluster = ReflectionTestUtils.getField(block, "positionInCluster");
-                assertThat(positionInCluster, is(equalTo(blockIndex)));
+                assertThat(positionInCluster).isEqualTo(blockIndex);
```

Comment:
This avoids the call to `ReflectionTestUtils.getField()`.
```suggestion
                assertThat(block).hasFieldOrPropertyWithValue("positionInCluster", blockIndex);
```